tests/lib/rules/match-any.ts

```diff
@@ -110,14 +110,35 @@ tester.run("match-any", rule as any, {
         },
         {
             code: "/[\\s\\S] [\\S\\s] [^] ./s",
-            output: "/. . . ./s",
+            output: "/. [\\S\\s] [^] ./s",
```

Comment:
I know that this is because of conflicts but it looks like a bug. Could you please add a comment here?

```suggestion
            // Only one character class gets fixed because they all depend on the `s` flag.
            // This shared dependency causes all of their fixes to conflict, so only one fix can be applied.
            output: "/. [\\S\\s] [^] ./s",
```

Reply:
Thank you!